easybuild/easyconfigs/e/EMAN2/EMAN2-2.21a-foss-2018a-Python-2.7.14.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'CMakeMake'
+
+name = 'EMAN2'
+version = '2.21a'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://blake.bcm.edu/emanwiki/EMAN2'
+description = """EMAN2 is the successor to EMAN1. It is a broadly based greyscale scientific image processing suite
+ with a primary focus on processing data from transmission electron microscopes. """
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['https://github.com/cryoem/eman2/archive']
+sources = ['v%(version)s.tar.gz']
+patches = ['EMAN2-2.21a.patch']
+checksums = [
+    '8a7310b60852220103a6676c7116f4119b290776de572a1510b982783bff46f7',  # v2.21a.tar.gz
+    '7399b4645c7fda87dc102f4ce3f7011764e40833984bb6cac025f840ed177a19',  # EMAN2-2.21a.patch
+]
+
+builddependencies = [('CMake', '3.10.2')]
+
+dependencies = [
+    ('Boost', '1.63.0', versionsuffix),
+    ('freetype', '2.9'),
+    ('FTGL', '2.1.3-rc5'),
+    ('GSL', '2.4'),
+    ('HDF5', '1.10.1'),
+    ('IPython', '5.7.0', versionsuffix),
+    ('libGLU', '9.0.0'),
+    ('libjpeg-turbo','1.5.3'),
+    ('LibTIFF', '4.0.9'),
+    ('libpng', '1.6.34'),
+    ('Mesa', '17.3.6'),
+    ('Python', '2.7.14'),
+    ('PyQt', '4.12.1', versionsuffix),
+]
+
+separate_build_dir = True
+
+preconfigopts = 'export CONDA_BUILD_STATE=BUILD && '
+preconfigopts += 'export PREFIX=%(installdir)s && '
+preconfigopts += 'export SP_DIR=%(installdir)s/lib/python%(pyshortver)s/site-packages && '
+preconfigopts += 'export FFTW3FDIR="$EBROOTFFTW" && '
+preconfigopts += 'export FFTW3DDIR="$EBROOTFFTW" && '
+preconfigopts += 'export HDF5DIR="$EBROOTHDF5" && '
+preconfigopts += 'export FTGLDIR="$EBROOTFTGL" && '
+
+sanity_check_paths = {
+    'files': [],
```

Comment:
@smoors Existing `EMAN2` easyconfig check for specific scripts in `bin`, why stop that?

Reply:
because there are many more files in `bin`, way too many to include, but I can add the existing ones back I you prefer.

Reply:
You don't need to check for all of them, but checking for a couple is definitely better than checking for now.